### DISCUSSION TITLE
Çember şeklinde ayin

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#215] [Zeka Sorusu]
Bir grup üyemiz çember şeklinde oturmuş soru ayini yapıyormuş. Beşinci üye ile ondokuzuncu üye karşı karşıya oturuyorsa bu çemberde kaç üyemiz vardır?

**Comment by KayraMertAydin** [comment]
eğer 5. üye ile 19. üye karşı karşıya oturuyorsa aralarında 6, 7, 8.... 17, 18 olmak üzere toplam 13 üye vardır. Diğer tarafta da 13 üye olmak zorundadır ancak 1, 2, 3, 4. üyeler zaten olduğu için 19dan ilerlemek için geriye 9 üye kalır. 19+9 = 28 olduğu için toplam 28 üye vardır.
